### PR TITLE
Updated this docs to be same as the current one

### PR DESCRIPTION
**Pull request by Suryansh5545**
Just updated doc file for readthedocs to be same as current one

### CODE REVIEW

**Comment by Suryansh5545** [comment]
#2560

File: docs/source/contribution.rst [hunk 1]
```diff
@@ -1,88 +1,66 @@
...
-pointers on how you can help.
+Thank you for your interest in contributing to EvalAI! Here are a few pointers about how you can help.
 
-Setting things up
-~~~~~~~~~~~~~~~~~
+### Setting things up
 
-To set up the development environment, follow the instructions in
-our README.
+To set up the development environment, follow the instructions in README.
```
Comment (gautamjajoo) [inline docs/source/contribution.rst]: Remove readme and add EvalAI docs section where the installation instructions are provided.

Comment (Suryansh5545) [reply docs/source/contribution.rst]: Done

**Comment by Suryansh5545** [comment]
**Here are the Screen Shots** 
Selection_003
Selection_006
Selection_005
Selection_004

**Pull request closed without merging**